scripts/microbe_stage/organelle.as

```diff
@@ -636,7 +648,7 @@ class PlacedOrganelle : SpeciesStoredOrganelleType{
 
         assert(organelleEntity == NULL_OBJECT, "PlacedOrganelle already had an entity");
 
-        const Float3 offset = organelle.calculateCenterOffset();
+        const Float3 offset = organelle.calculateModelOffset();
```

Comment:
I think this is used in one place. If that is the case, this should be moved to be just before that line.

Reply:
did it